packages/@tinacms/schema-tools/src/schema/TinaSchema.ts

```diff
@@ -452,14 +452,15 @@ export class TinaSchema {
         ? this.getCollection(collectionOrString)
         : collectionOrString
     const normalPath = normalizePath(collection.path)
+    const slash = normalPath ? '/' : ''
```

Comment:
I don't know if I follow why `slash` may be an empty string?

Reply:
If normalPath is `""` slash is empty otherwise its a /. I will make a better name for this.

Reply:
Updated